assets/sass/_aesop.scss

```diff
@@ -40,7 +40,7 @@
         padding: 20px 0;
       }
 
-      @media screen and (max-width: $width-phone) {
+      @include max-width($width-phone) {
```

Comment:
Expected item on line 43 to appear before line 25. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets